Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityCameraProfileInspector.cs

```diff
@@ -1,99 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information. 
-
-using Microsoft.MixedReality.Toolkit.Utilities.Editor;
-using UnityEditor;
-using UnityEngine;
-
-namespace Microsoft.MixedReality.Toolkit.Editor
-{
-    [CustomEditor(typeof(MixedRealityCameraProfile))]
-    public class MixedRealityCameraProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
```

Comment:
I didn't see this called out in the PR notes - was there some other camera profile inspector that already existed (so this was a duplicate?). Otherwise it looks like if you view the profile itself (outside of the MRTK object) you'll lose this functionality